packages/app/cypress/e2e/specs_list_e2e.cy.ts

```diff
@@ -0,0 +1,88 @@
+describe('App: Spec List (E2E)', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer('e2e')
+    cy.visitApp()
+  })
+
+  it('shows the "Specs" navigation as highlighted in the lefthand nav bar', () => {
+    cy.findByLabelText('Sidebar').within(() => {
+      cy.findByText('Specs').should('be.visible')
+      cy.findByText('Specs').click()
+    })
+
+    cy.get('.router-link-active').findByText('Specs').should('be.visible')
+  })
+
+  it('displays the App Top Nav', () => {
+    cy.get('[data-cy="app-header-bar"]').should('be.visible')
+    cy.get('[data-cy="app-header-bar"]').findByText('Specs').should('be.visible')
+  })
+
+  it('shows the "E2E Specs" label as the header for the Spec Name column', () => {
+    cy.get('[data-cy="specs-testing-type-header"]').should('contain', 'E2E Specs')
+  })
+
+  it('allows you to search and filter the list of specs in the list', () => {
```

Comment:
Could we maybe update the test project to have more than one test? So that we can test "no filter", "filtered down to less than the # of tests", "filtered down to no tests"